CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ add_subdirectory(Map/Test)
 include_directories(Combat/Test)
 add_subdirectory(Combat/Test)
 
-add_library(source_objects_lib OBJECT ${SOURCES})
+add_library(source_objects_lib OBJECT ${SOURCES} Map/Source/Map.cpp)
```

Comment:
To jest zbędnę. Map.cpp powinna znajdować się w zmiennej SOURCES, wiec robisz dublacje